Library/Homebrew/vendor/README.md

```diff
@@ -0,0 +1,56 @@
+Vendored Dependencies
+=====================
+
+* okjson is a pure-Ruby JSON parser written by
+[Keith Rarick](https://github.com/kr). Its upstream repository can be found
+[here](https://github.com/kr/okjson).
+
+* ruby-macho is a pure-Ruby Mach-O parser written by
+[William Woodruff](https://github.com/woodruffw). Its upstream repository can
+be found [here](https://github.com/woodruffw/ruby-macho).
```

Comment:
Alright!
By the way, okjson is now on revision 45. It looks like a few numerical parsing issues were fixed in the last few revisions, so it might be worth updating the vendored version at some point.